deploy/scripts/lhexec

```diff
@@ -45,7 +47,9 @@ exec_command() {
     ENGINE_PORT_FILTER="{.items[?(@.spec.volumeName==\"${VOLUME_NAME}\")].status.port}"
     ENGINE_PORT=`kubectl -n ${NS} get lhe --output=jsonpath="${ENGINE_PORT_FILTER}"`
 
-    kubectl -n ${NS} exec -it ${INSTANCE_MANAGER_NAME} -- bash -c "longhorn --url localhost:${ENGINE_PORT} ${COMMAND_ARGS}"
+	LONGHORN_BIN_PATH=`kubectl -n ${NS} exec -it ${INSTANCE_MANAGER_NAME} -- bash -c "ps -eo command | grep "${VOLUME_NAME}" | grep -v grep | awk '{ printf(\"%s\", \\$1)}'"`
```

Comment:
`grep` like this can get both `test` and `testabc` for `test`.

Reply:
fixed